Code/GraphMol/FMCS/FMCS.cpp

```diff
@@ -236,6 +238,24 @@ bool checkAtomChirality(const MCSAtomCompareParameters& p,
   return true;
 }
 
+bool checkAtomDistance(const MCSAtomCompareParameters& p,
+                       const ROMol& mol1, unsigned int atom1,
+                       const ROMol& mol2, unsigned int atom2){
+  unsigned int confId1 = 0;
```

Comment:
The normal way to handle unspecified conformers is using a confId of -1. That way the default conformer (not necessarily the one with Id 0) is used.

Reply:
Thanks